back/spec/models/notification_spec.rb

```diff
@@ -86,81 +86,6 @@
-        _other_user = create(:admin)
-        _other_admin = create(:user, email: 'user2@example.com')
-
-        activity = create(:activity, item: phase, action: 'started')
-
-        notifications = Notifications::ProjectPhaseStarted.make_notifications_on(activity)
-        expect(notifications.map(&:recipient_id)).to match_array [user.id]
-      end
-
-      context 'when the project is visible only to some groups' do
-        let(:phase) { create(:active_phase) }
-        let!(:project) do
-          phase.project.tap do |project|
-            project.visible_to = 'groups'
-            project.groups << create(:group)
-          end
-        end
-        let(:notifications) do
-          activity = create(:activity, item: phase, action: 'started')
-          Notifications::ProjectPhaseStarted.make_notifications_on(activity)
-        end
-
-        context 'and the user moderates the project' do
-          before { create(:project_moderator, projects: [project], manual_groups: [project.groups.first]) }
-
-          it { expect(notifications.map(&:recipient_id)).to eq [] }
-        end
-
-        context 'and the user moderates another project' do
-          let!(:user) { create(:project_moderator, manual_groups: [project.groups.first]) }
-
-          it { expect(notifications.map(&:recipient_id)).to eq [user.id] }
-        end
-      end
-    end
-
-    describe 'project_phase_upcoming' do
-      let(:project) { create(:project_with_current_phase) }
-      let(:other_project) { create(:project_with_current_phase) }
-
-      let(:project_folder) { create(:project_folder, projects: [project]) }
-      let(:other_project_folder) { create(:project_folder, projects: [other_project]) }
-
-      let!(:regular_user) { create(:user) }
-      let!(:moderator_of_project) { create(:project_moderator, projects: [project]) }
-      let!(:moderator_of_other_project) { create(:project_moderator, projects: [other_project]) }
-      let!(:moderator_of_project_folder) { create(:project_folder_moderator, project_folders: [project_folder]) }
-      let!(:moderator_of_other_project_folder) { create(:project_folder_moderator, project_folders: [other_project_folder]) }
-      let!(:admin) { create(:admin) }
-
-      it 'notifies all admins and only relevant moderators' do
-        activity = create(:activity, item: project.phases[2], action: 'upcoming')
-        notifications = Notifications::ProjectPhaseUpcoming.make_notifications_on activity
-        notification_ids = notifications.pluck(:recipient_id)
-
-        expect(notification_ids).to include moderator_of_project.id
-        expect(notification_ids).to include moderator_of_project_folder.id
-        expect(notification_ids).to include admin.id
-        expect(notification_ids).not_to include regular_user.id
-        expect(notification_ids).not_to include moderator_of_other_project.id
-        expect(notification_ids).not_to include moderator_of_other_project_folder.id
-      end
-    end
```

Comment:
Removed from this file, as I forgot to do this in #3366 (when I moved the test to `project_phase_upcoming_spec.rb`, but did not remove the original here)